server/etcdserver/apply/apply.go

```diff
@@ -231,12 +232,14 @@ func (a *applierV3backend) Alarm(ar *pb.AlarmRequest) (*pb.AlarmResponse, error)
 			break
 		}
 		resp.Alarms = append(resp.Alarms, m)
+		alarms.With(prometheus.Labels{"server_id": types.ID(ar.MemberID).String(), "alarm_type": m.Alarm.String()}).Inc()
```

Comment:
This isn't an issue, but it seems `WithLabelValues` is better than `prometheus.Labels`, because we do not need to type the hard coded labels everywhere. 

```
alarms.WithLabelValues(types.ID(ar.MemberID).String(), m.Alarm.String()).Inc()
```

I searched the repo, and majorities are using `WithLabelValues`, and only 6 places are using `prometheus.Labels`. So I suggest to use `WithLabelValues` so as to be consistent.